### ISSUE TITLE
Elastisearch - which set of filters are being used?

### ISSUE THREAD

**Issue by MNSmith** [BiologicalRecordsCentre/iRecord#740]
Two issues:

1) When using the old Verification screen, I am finding the list of "standard" filters is different from those displayed on the new ES screen.  As far as I can see, ES appears to be using the list of filters that one gets when one uses the "Explore all records" screen, including custom ones I have made there (see images)  Some of the "standard" filters from the old Grid appear not to be there by default in the ES and any custom filters i made in the old grid will have to be recreated in the ES screen.  Could we just not have one list of filters per username please?

Old Grid List
![Image1](https://user-images.githubusercontent.com/18451743/71837506-7b578f00-30ae-11ea-887b-0f115d55ecd1.jpg)

Explore records page filter list
![image 2](https://user-images.githubusercontent.com/18451743/71837561-a346f280-30ae-11ea-8fd7-b3f836e5bb30.jpg)

ES filter list
![Image2](https://user-images.githubusercontent.com/18451743/71837522-86122400-30ae-11ea-8b0a-e5dc02c7cc57.jpg)

2) In Elastisearch, creating a filter to return "Queried" items returns an empty grid.  Lots of records listed if I use the default "Queried" filter on the old grid page.

**Comment by kitenetter** [comment]
The queried filter on the new page seems to be working for me, let me know @MNSmith if it isn't for you.

Agree that we need to clarify the filter behaviour.

**Comment by johnvanbreda** [comment]
I suspect the new Elasticsearch filter dropdown loads filters for the reporting task, rather than verification task. This needs changing.

**Comment by MNSmith** [comment]
I am still seeing the two different lists of filters on the old and new verifcation grids.

Using a "Queried only" filter on the old page returns 473 records.
Using a "Queried only " filter on the new ES page returns 0 records - so there is a problem here.

**Comment by kitenetter** [comment]
Some further observations:

**From "Verify - new"**
On the new verification page, I've created a new (taxon-based) filter using the "Create a filter" button. I named and saved the filter.
1. There was no message to say that it had been successfully saved, but the filter **does** now appear in my list of filters for the new verification page.
2. The new filter **does not** appear in the list of filters for the old verification page, nor as a filter option for a download.

**From "Verify"**
I wanted to create an equivalent filter on the old page to see if that would behave in the same way, but it appears that taxon filtering is no longer available from the Create a filter window.

**Comment by johnvanbreda** [comment]
@kitenetter I've changed the sharing mode for the loaded filters to Verification on the ES verification page so you get the same set. I also tried saving a filter on the new page and the filter saved message did appear, so might be worth double checking that. 
I'll close this issue now as I think the main point is covered - there are a couple of other points made here that might already be covered elsewhere (e.g. mismatch of record counts).
